### PR TITLE
[tests-only] prepare required resources rather than using skeleton directories in API WebDav move, operations and preview features 

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- skeleton file/folders are used more wisely on the following features suite
  - apiWebdavMove1
  - apiWebdavMove2
  - apiWebdavOperations
  - apiWebdavPreviews

## Related Issue

- Part of https://github.com/owncloud/QA/issues/658

## How Has This Been Tested?

- :robot

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by dpakach — COMMENTED** [review]
lgtm :+1:

File: tests/acceptance/features/apiWebdavMove1/moveFileAsync.feature [hunk 1]
```diff
@@ -241,9 +252,11 @@ Feature: move (rename) file
     #need to slowdown the request for longer than the timeout
     #when doing LazyOps the server does not close the connection
     #so we timout the request and chech the job-status
-    Given the HTTP-Request-timeout is set to 5 seconds
+    Given user "Alice" has uploaded file "filesForUpload/textfile.txt" to "fileToMove.txt"
+    And the HTTP-Request-timeout is set to 5 seconds
     And the MOVE dav requests are slowed down by 10 seconds
-    When user "Alice" moves file "/welcome.txt" asynchronously to "/textfile0.txt" using the WebDAV API
+
```
Comment (individual-it) [inline tests/acceptance/features/apiWebdavMove1/moveFileAsync.feature:258]: extra line

File: tests/acceptance/features/apiWebdavOperations/search.feature [hunk 2]
```diff
@@ -46,11 +46,10 @@ Feature: Search
     Given using <dav_version> DAV path
     When user "Alice" searches for "ol" using the WebDAV API
     Then the HTTP status code should be "207"
-    And the search result should contain "4" entries
+    And the search result should contain "3" entries
     And the search result of user "Alice" should contain these entries:
       | /just-a-folder           |
       | /upload folder           |
-      | /FOLDER                  |
```
Comment (individual-it) [inline tests/acceptance/features/apiWebdavOperations/search.feature:53]: lets create `FOLDER` also, that is a good test when searching for `ol`

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by jasson99 — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiWebdavOperations/search.feature (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavOperations/search.feature:53]: okay!

File: tests/acceptance/features/apiWebdavMove1/moveFileAsync.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavMove1/moveFileAsync.feature:258]: removed!

**Review by individual-it — APPROVED** [review]

**Pull request merged**